### PR TITLE
Update CHANGELOG.md in Preparation for 24 Release

### PR DESCRIPTION
**Pull request by jesscall**
## Brief summary of changes
Updating CHANGELOG.md in preparation for 24.0 release 

* Resolves @ridz1208 's anguish

### CODE REVIEW

**Comment by ridz1208** [comment]
@jesscall it really does haha

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +40,8 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 - Consents may now be grouped in UI of consent tab (PR #6042, PR #6044)
 #### API Documentation (**New Module**)
 - New module mostly intended for developers, this module provides a user interface to inspect and try LORIS modules API.
+#### Data Dictionary
+- Changed filter to multiselect  (PR #7040)
```
Comment (ridz1208) [inline CHANGELOG.md:44]: changed what filter ?

Comment (jesscall) [reply CHANGELOG.md:44]:
```suggestion
- Changed instrument filter to multiselect  (PR #7040)
```

**Review by ridz1208 — APPROVED** [review]

**Comment by ridz1208** [comment]
@driusan all yours

**Comment by driusan** [comment]
@jesscall can't merge because of conflicts.

**Comment by ridz1208** [comment]
@driusan I'm on it

**Comment by ridz1208** [comment]
@driusan done

**Pull request merged**